kotlintest-assertions/src/main/kotlin/io/kotlintest/properties/Gen.kt

```diff
@@ -363,6 +380,33 @@ interface Gen<T> {
       override fun random(): Sequence<UUID> = generateSequence { UUID.randomUUID() }
     }
 
+    fun localDate(minYear: Int = 1, maxYear: Int = 2030): Gen<LocalDate> = object : Gen<LocalDate> {
+      override fun constants(): Iterable<LocalDate> = emptyList()
+      override fun random(): Sequence<LocalDate> = generateSequence { LocalDate.of(RANDOM.nextInt(maxYear - minYear) + minYear, RANDOM.nextInt(12) + 1, RANDOM.nextInt(31) + 1) }
+    }
+
+    fun localTime(): Gen<LocalTime> = object : Gen<LocalTime> {
+      override fun constants(): Iterable<LocalTime> = emptyList()
+      override fun random(): Sequence<LocalTime> = generateSequence { LocalTime.of(RANDOM.nextInt(24), RANDOM.nextInt(60), RANDOM.nextInt(60)) }
+    }
+
+    fun localDateTime(minYear: Int = 1, maxYear: Int = 2030): Gen<LocalDateTime> = object : Gen<LocalDateTime> {
+      override fun constants(): Iterable<LocalDateTime> = emptyList()
+      override fun random(): Sequence<LocalDateTime> = generateSequence { LocalDateTime.of(RANDOM.nextInt(maxYear - minYear) + minYear, RANDOM.nextInt(12) + 1, RANDOM.nextInt(31) + 1, RANDOM.nextInt(24), RANDOM.nextInt(60), RANDOM.nextInt(60)) }
```

Comment:
Maybe better to use the other generators here. `generatedLocalDate.atTime(generatedTime)`

Reply:
Prevents extra maintance points (although unlikely)